Dockerfile

```diff
@@ -86,10 +86,10 @@ RUN apt-get clean
 RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Install crontab to setup jobs
-RUN echo "0 0 * * * /usr/bin/python /code/manage.py generate_tree" >> /code/cronjob
-RUN echo "0 * * * Mon /usr/bin/python /code/manage.py reset_container_limits" >> /code/cronjob
-RUN echo "0 1 * * * /bin/bash /code/scripts/backup_db.sh" >> /code/cronjob
-RUN echo "0 2 * * * /usr/bin/python /code/manage.py cleanup_dummy" >> /code/cronjob
+RUN echo "0 0 * * * /usr/local/bin/python /code/manage.py generate_tree > /var/log/generate_tree.log 2>&1 " >> /code/cronjob
+RUN echo "0 * * * Mon /usr/local/bin/python /code/manage.py reset_container_limits > /var/log/reset_container_limits.log 2>&1 " >> /code/cronjob
```

Comment:
so if you like, you can take out this line to generate_tree. If not, I'll remove for a separate PR. I implemented the comparison tree when I was very idealistic about making fun interfaces, and I think it goes out of scope for the purpose of a registry. At least for now - if it's wanted back we can re-implement something (that doesn't require an external dependency).

Reply:
Done.